devices/Sim7080/SIM7080.nuspec

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+	<metadata>
+		<id>nanoFramework.Iot.Device.SIM7080</id>
```

Comment:
namewise, we're going for Sim7080 following Pascal notation. Please adjust in the nuspec as well

Reply:
Changed to Pascal notation